website/views/pages/homepage.ejs

```diff
@@ -144,7 +156,7 @@
   </div>
   <div id="testimonials" style="background: #F4F6FA; position: relative">
     <div class="container-lg text-center justify-content-center py-5 mx-auto">
-      <h3 class="px-4 mt-5 mb-3">Join the open-source community</h3>
+      <h3 class="px-4 mt-5 mb-3">Fleet is open source and community supported.</h3>
       <p style="max-width: 860px" class="px-3 px-xs-5 px-md-4 mx-auto">
         Software engineers. Security specialists. IT administrators. Lots of professionals are working together to make Fleet even better. From contributing patches to dreaming up features.
```

Comment:
@mikermcneil, we decided to keep this line. We felt that it better conveyed the community message.

cc @chris-mcgillicuddy